src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -458,6 +462,7 @@ export class InstructorSessionsPageComponent extends InstructorSessionModalPageC
           this.sessionsTableRowModels.push(m);
           this.updateInstructorPrivilege(m);
           this.statusMessageService.showSuccessToast('The feedback session has been restored.');
+
```

Comment:
Unnecessary line change here

Reply:
I have removed the Unnecessary line change. It is difficult to capture the change in a screenshot so I used a video to demo the changes instead.

Here is the link to the video:
(https://drive.google.com/file/d/1Q4soInj7UlcOhUuodINzNfnxtXai4-vO/view?usp=sharing)